<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputPopUp</class>
 <widget class="QDialog" name="InputPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="inputWidgets" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>221</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="maxLength">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitBtn">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
